packages/codex-context/src/backend.ts

```
@@ -45,9 +45,12 @@
 
 export class OllamaBackend implements BackendClient {
-  private log = createLogger("codex-context", {
-    component: "backend",
-    driver: "ollama",
+  private log = createLogger({
+    service: "codex-context",
+    base: {
+      component: "backend",
+      driver: "ollama",
+    },
   });
```

ESLint

[error] Property should have a readonly modifier.

(functional/prefer-immutable-types)